src/utils/configs.ts

```diff
@@ -1,19 +1,27 @@
 export const tsconfig = {
   compilerOptions: {
-    target: "es6",
-    lib: ["es6", "dom"],
-    outDir: "build",
-    rootDir: ".",
-    strict: true,
-    noImplicitAny: true,
-    esModuleInterop: true,
-    resolveJsonModule: true,
-    allowJs: true,
-    types: ["node"]
+      target: "ES2020",
+      module: "commonjs",
```

Comment:
This option should not be used. In this case, the code generated will be CommonJS. We don't want that as some libraries can't be imported from this type of code.

```suggestion
      module: "NodeNext",
```